README.md

```diff
@@ -1,34 +1,167 @@
+- Configure the ANDROID_HOME environment variable
+- Using a virtual device (under Preparing the Android device)
+
+#### Homebrew
+
+Follow the “Install Homebrew” instructions on [Homebrew’s website](https://brew.sh).
+
+After running the installer, follow these instructions:
+
+```
+Warning: /opt/homebrew/bin is not in your PATH.
+  Instructions on how to configure your shell for Homebrew
+  can be found in the 'Next steps' section below.
+
+==> Next steps:
+- Run this command in your terminal to add Homebrew to your PATH:
+    eval "$(/opt/homebrew/bin/brew shellenv)"
+```
+
+#### Watchman
+
+Use Homebrew to install Watchman:
+
+```
+brew install watchman
+```
+
+#### Java Development Kit
+
+Install the Java Development Kit with these commands:
+
+```
+brew tap homebrew/cask-versions
+brew install --cask zulu17
+
+# Get path to where cask was installed to double-click installer
+brew info --cask zulu17
+```
+
+> After you install the JDK, update your JAVA_HOME environment variable. If you used above steps, JDK will likely be at /Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home
 
-Prerequisites: Node >20, NPM
+#### Install Android Studio
+
+Follow the instructions in the quickstart guide.
+
+#### Install the Android SDK
+
+Follow the instructions in the quickstart guide.
+
+#### Configure the ANDROID_HOME environment variable
+
+Follow the instructions in the quickstart guide.
+
+#### Using a virtual device (under Preparing the Android device)
+
+Follow the instructions in the quickstart guide.
+
+### Windows
+
+Follow the [React Native CLI Quickstart](https://reactnative.dev/docs/environment-setup?guide=native&os=windows&platform=android) instructions for your operating system.
+
+Be sure to follow the quickstart instructions for:
+
+- JDK
+- Install Android Studio
+- Install the Android SDK
+- Configure the ANDROID_HOME environment variable
+- Add platform-tools to Path
+- Using a virtual device (under Preparing the Android device)
+
+#### JDK
+
+Follow the instructions in the quickstart guide.
+
+#### Install Android Studio
+
+Follow the instructions in the quickstart guide.
+
+#### Install the Android SDK
+
+Follow the instructions in the quickstart guide.
+
+#### Configure the ANDROID_HOME environment variable
+
+Follow the instructions in the quickstart guide.
+
+#### Add platform-tools to Path
+
+Follow the instructions in the quickstart guide.
+
+#### Using a virtual device (under Preparing the Android device)
+
+Follow the instructions in the quickstart guide.
 
 ### Install dependencies
 
+Clone this repo and run:
+
 ```
-npm install
+npm ci
 ```
 
-### Install Expo Go
+## Running the app
 
-Install Expo Go on your mobile device from the appropriate app store. This will allow you to quickly run demos you find online with your mobile device. This app will not run in Expo Go because of OAuth, but many other apps will so its a useful tool to be aware of.
+Build and launch the app in the Android Emulator with:
```

Comment:
I would hint, or inform the user that they need the android studio emulator running before 'npm run android'

Reply:
It’s here:

>  - Using a virtual device (under Preparing the Android device)

Point taken though, that should probably be moved from the Setup instructions to the commands for running.